examples/demo.rs

```diff
@@ -22,6 +24,7 @@ impl mq::EventHandler for Stage {
     fn update(&mut self, _ctx: &mut mq::Context) {}
 
     fn draw(&mut self, ctx: &mut mq::Context) {
+        self.egui_mq.egui_input().pixels_per_point = Some(self.scale.min(2.0).max(0.7));
```

Comment:
```suggestion
        self.egui_mq.egui_input().pixels_per_point = Some(self.scale.clamp(0.7, 2.0));
```

…but it is better to do the clamping when you modify the scale

Reply:
done